internal/core/core.go

```diff
@@ -227,59 +226,6 @@ func GetInterpolationMethod(interpolation string) (int, error) {
 	}
 }
 
-func GetAttributeType(attribute string) (int, error) {
```

Comment:
Maybe we should leave it in core? This is the only one "translation" method that gets moved out. Not regular surface or bounds.

So maybe it would be easier to find here?

Reply:
Sure